httpclient-vertx/src/main/java/io/fabric8/kubernetes/client/vertx/VertxHttpClientBuilder.java

```diff
@@ -136,6 +136,13 @@ public SslContextFactory sslContextFactory() {
         }
       });
     }
+
+    // Since the factory is not explicitly typed as VertxHttpClientFactory, we need to check the type before invoking
```

Comment:
It's perfectly fine to do this instanceOf check. Quarkus is using their own factory type with this builder.

Reply:
An alternative to make this available to other factories is to introduce an interface for this specific additionalConfig method signature.